pvlib/bifacial/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+The ``bifacial`` module contains functions to model irradiance for bifacial
+modules.
+
+"""
+
+from pvlib.bifacial import pvfactors, infinite_sheds, utils   # noqa: F401
```

Comment:
> * Can `infinite_sheds.csv` be removed?  As far as I can tell it's not used for anything.

Yes. This is output of an Excel version that @mikofski has, but the algorithm here has deviated from what was done in the spreadsheet and so `infinite_sheds.csv` doesn't provide known output.

> * Doesn't the term "infinite sheds" apply equally well to the other popular 2D VF models?  I wonder if we should call this model something more specific.

Open for suggestions. This code isn't exactly what is done in the PVSC paper, because the published technique doesn't handle horizontal arrays, and this code doesn't use averaged view factors at one step (I don't recall which).

> A comment about the model itself: the assumption of isotropic sky diffuse irradiance seems a little regrettable for this level of model detail. Especially for tracking systems, the difference between isotropic and anisotropic models for front-side sky diffuse can be quite large, even roughly the same magnitude as the entire rear-side irradiance:

Fair point. And an excellent illustration of the value of hosting multiple bifacial irradiance models in pvlib.

Any model that is computing incoming sky diffuse using the difference of cosine view factor is implicitly assuming that the incoming irradiance is isotropic. The alternative (separating sky diffuse into circumsolar, horizon and rest-of-sky) requires view factors that are aware of elevation angle, and a different interface (e.g. `sky_diffuse_horizon` would be an input, or the code would have to provide a selection of diffuse POA models). Neither is appealing here, either for scope, or to not depart farther from the reference than we already have.

The differences you plot between `infinite_sheds` and `pvfactors` could also be affected algorithmic differences: `pvfactors` is accounting for reflections from all nearby surfaces, where `infinite_sheds` only accounts for ground reflections.

There is also an error in the code which I need to diagnose and fix - see the failed test in the latest push.